src/nvim/eval.c

```diff
@@ -10583,6 +10583,7 @@ static void f_has(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     "linebreak",
     "lispindent",
     "listcmds",
+    "liveupdate",
```

Comment:
Normally we don't add named features for new nvim-specific features, as nvim will always be compiled with all features (with a few exceptions). Instead a plugin should check for the thing the feature adds, an api function in this case. This is possible in principle by inspecting the api metadata, but maybe we should make this simpler by overloading `exist()` (`exist('+')` does not work as not all api functions are vimL functions, like this won't be).